docs/conf.py

```
@@ -309,7 +311,7 @@
 myst_gfm_only = False
 
 myst_html_meta = {
-	"github_url": f"https://github.com/reactive-firewall/{project}"
+	"github_url": sanitize_url(f"https://github.com/reactive-firewall/{project}")
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)